### ISSUE TITLE
Failure with dev httr

### ISSUE THREAD

**Issue by hadley** [USGS-R/geoknife#222]
Probably because `content()` now defaults to using xml2, not XML.

```
checking examples ... ERROR
Running examples in ‘geoknife-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: XML
> ### Title: XML from set of objects
> ### Aliases: XML XML,ANY,webdata,webprocess-method XML,webgeom-method
> 
> ### ** Examples
> 
> wd <- webdata('prism',times = as.POSIXct(c('2001-01-01','2002-02-05')))
> wg <- webgeom('state::Wisconsin')
Error: Please install xml2 package
Execution halted
```
```
checking tests ... ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  16: initialize(value, ...)
  17: .local(.Object, ...)
  18: `values<-`(`*tmp*`, value = "09020306")
  19: `values<-`(`*tmp*`, value = "09020306")
  20: fetchGML_IDs(.Object)
  21: content(response)
  22: parse_auto(raw, type, encoding, ...) at /Users/hadley/Documents/web/httr/R/content.r:81
  23: parser(content, type = type, encoding = encoding, ...) at /Users/hadley/Documents/web/httr/R/content-parse.r:45
  24: need_package("xml2") at /Users/hadley/Documents/web/httr/R/content-parse.r:105
  25: stop("Please install ", pkg, " package", call. = FALSE) at /Users/hadley/Documents/web/httr/R/utils.r:41
  
  Error: Please install xml2 package
  Execution halted
```
```
checking re-building of vignette outputs ... NOTE
Error in re-building vignettes:
  ...
Quitting from lines 91-93 (geoknife.Rmd) 
Error: processing vignette 'geoknife.Rmd' failed with diagnostics:
no applicable method for 'xpathApply' applied to an object of class "c('xml_document', 'xml_node')"
Execution halted

```

**Comment by jread-usgs** [comment]
Thank you @hadley I will update and set the minimum version of `httr`.

**Comment by jread-usgs** [comment]
Resolved via #223
